qiskit/transpiler/passes/optimization/consolidate_blocks.py

```diff
@@ -71,6 +78,7 @@ def __init__(
             )
         else:
             self.decomposer = TwoQubitBasisDecomposer(CXGate())
+        self._approximation_degree = approximation_degree
```

Comment:
I don't think we need to store this; decomposers by their nature need to be re-entrant and so are safe for re-use.

Reply:
Again, that is cruft from the first version of the PR; it's not used anywhere now.

Reply:
Done in 248479d